packages/graphiql/package.json

```diff
@@ -17,37 +17,39 @@
   },
   "license": "MIT",
   "main": "dist/index.js",
-  "module": "esm/index.js",
+  "module": "dist/index.mjs",
   "types": "dist/index.d.ts",
   "files": [
     "dist",
-    "esm",
-    "src",
     "graphiql.js",
     "graphiql.min.js",
     "graphiql.min.js.map",
     "graphiql.css",
-    "graphiql.min.css",
-    "graphiql.min.css.map"
+    "graphiql.min.css"
   ],
+  "exports": {
+    "./package.json": "./package.json",
+    "./style.css": "./dist/style.css",
+    "./graphiql.css": "./dist/style.css",
```

Comment:
exports don'tt work with jsdelivr/unpkg unfortunately, we must publish to npm with these exact files

Reply:
I added `prepublishOnly` with exact files https://github.com/graphql/graphiql/blob/ad75cdaacd0f0043dce436d6869299af0a618094/packages/graphiql/package.json#L48